src/com/janek/basics/Counting.java

```diff
@@ -0,0 +1,36 @@
+package com.janek.basics;
+
+import static java.lang.Integer.parseInt;
+import static java.lang.String.format;
+import static java.lang.System.*;
+
+public class Counting {
+    public static void main(String[] args) {
```

Comment:
Sorry, I don't get it. 
You had to write a function that will calculate the sum of decimal numbers from 1 to x.
Your code doesn't do it